src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,111 @@
+package ui;
+
+
+public class Ui {
+    String line = "~~*~~*~~*~~*~~*~~*~~*~~*~~*~~\n";
```

Comment:
Already declared above in TaskList.java. Should be reused there. Also should be a constant, so SCREAMING_SNAKE_CASE should be used.